static/docs/get-started/metrics.md

```diff
@@ -1,6 +1,6 @@
 # Experiment Metrics
 
-The last stage we would like to add to the pipeline is the evaluation stage.
+The last stage we would like to add to our pipeline is its accuracy evaluation.
```

Comment:
Accuracy has a very specific meaning. It's one of many metrics that are possible. I would keep it as this.

Reply:
Fixed